packages/cli/src/templates/query.stub

```diff
@@ -0,0 +1,18 @@
+{{#imports}}
+import { {{commaSeparatedComponents}} } from '{{{packagePath}}}'
+{{/imports}}
+
+@Query({
+  authorize: // Specify authorized roles here. Use 'all' to authorize anyone
+})
+export class {{{ name }}} {
+  public constructor(
+    {{#fields}}
+    readonly {{{name}}}: {{{type}}},
+    {{/fields}}
+  ) {}
+
+  public static async handle(query: {{{ name }}} , register: Register): Promise<void> {
+    register.events( /* YOUR EVENT HERE */)
```

Comment:
The body should be other thing. Maybe just a comment saying "your query code here"

Reply:
Yep, copy/paste issue